Source/Core/LMCHFSSResponseFileHandler.cc

```diff
@@ -108,14 +114,24 @@ namespace locust
             fTFComplex[i][0]=tfArray.at(i).real();
             fTFComplex[i][1]=tfArray.at(i).imag();
         }
-        fComplexFFT.SetupIFFT(fTFNBins,fInitialTFIndex,fTFBinWidth);
-	fFIRNBins=fTFNBins+2*fComplexFFT.GetShiftNBins();
+	double TFBinWidth = 2./9.*fInitialTFIndex/(1.0*fTFNBins); //Calculates TFBinWidth assuming frequency range defined as .9 to 1.1 the center of the frequency bins.
```

Comment:
I don't really understand the explanation in the comment. Could you make it a bit clearer?

Reply:
I realized that it was a little confusing having both a `TFBinWidth` and `fTFBinWidth` variable, so I renamed and moved one to `AnalyticTFBinWidth` to more clearly show the differentiate them. I've also updated the comments to (hopefully) be more clear.

	if(GeneratedTF){ 
		//if TF generated based on config file (frequency ranges from 0.9 - 1.1 times the center given in .json config file), calculate the TF bin width given number of bins (also set in .json config file)).
		double AnalyticTFBinWidth = 2./9.*fInitialTFIndex/(1.0*fTFNBins);
		fComplexFFT.SetupIFFT(fTFNBins,fInitialTFIndex,AnalyticTFBinWidth);//Uses binwidth as calculated in the previous line based on internally generated TF
	}
	else{ 
		//if TF read from externally generated TF file, use TF bin width as given in .json config file
		fComplexFFT.SetupIFFT(fTFNBins,fInitialTFIndex,fTFBinWidth); //Uses degenerately defined fTFNBins AND fTFBinWidth if using external TF function.
	}

Reply:
I don't understand the logic here. 
Why is the `bin width = 2/(9*initialindex*nBins)` ?